code/modules/clothing/under/accessories.dm

```diff
@@ -197,6 +197,11 @@
 	medaltype = "medal-gold"
 	custom_materials = list(/datum/material/gold=1000)
 
+/obj/item/clothing/accessory/medal/gold/bureaucracy
+	name = "Excellence in Bureaucracy Medal"
+	desc = "Awarded for services to the Corporation in management"
```

Comment:
should this have a period at the end?

Reply:
Yes I will do a grammar pass this evening.